src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -247,6 +249,7 @@ public JobModel mapFlightStateToJobModel(FlightState flightState) {
 
     return new JobModel()
         .id(flightState.getFlightId())
+        .className(flightState.getClassName())
```

Comment:
This is fine for now, but I'd love "Human-readable names" for flights as well.

Reply:
Yeah, I think that's probably what description is for I think.